src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -557,7 +557,8 @@ namespace QgsWfs
         }
 
         // each Feature requested by FEATUREID can have each own property list
-        QString key = QStringLiteral( "%1(%2)" ).arg( typeName, propertyName );
+        // use semicolon that is not in typename
```

Comment:
1. `:` is not a semicolon
2. are we sure this is not in typename?

Reply:
I am sure it is not in typename, because typename can be used as XML tag name and because @elpaso fixed a bug by replacing colon in typenames https://github.com/qgis/QGIS/commit/65b272d1dcc6ffdcf8add04d4d0d11228b231229 that fixes #41292